bin/rspecq

```diff
@@ -95,19 +107,22 @@ opts[:max_requeues] ||= Integer(ENV["RSPECQ_MAX_REQUEUES"] || DEFAULT_MAX_REQUEU
 raise OptionParser::MissingArgument.new(:build) if opts[:build].nil?
 raise OptionParser::MissingArgument.new(:worker) if !opts[:report] && opts[:worker].nil?
 
+redis_opts = {host: opts[:redis_host]}
+redis_opts[:url] = opts[:redis_url] if opts[:redis_url]
```

Comment:
This should also be settable via an env variable (e.g. `RSPECQ_REDIS_URL`), in the same manner the other config opts are set.